core/src/main/scala/org/broadinstitute/dsde/rawls/spendreporting/SpendReportingService.scala

```diff
@@ -358,7 +358,8 @@ class SpendReportingService(
   }
 
   def getAllUserWorkspaceQuery(
-    billingProjects: Map[BillingProjectSpendExport, Seq[(GoogleProjectId, WorkspaceName)]],
+    billingProject: BillingProjectSpendExport,
+    workspaces: Seq[(GoogleProjectId, WorkspaceName)],
```

Comment:
the `WorkspaceName`s aren't used anywhere inside this method. If you changed this method to accept a `Seq[GoogleProjectId]` instead then some interior code gets a little simpler